### PR TITLE
SIMD has been removed from stage 3

### PR DESCRIPTION
**Pull request by krainboltgreene**
According to the readme for the proposal this has been removed? This document should reflect the current state and not the last known state, correct?

### CODE REVIEW

**Comment by ljharb** [comment]
This repo is the source of truth, not individual proposal readmes.

The [notes](https://github.com/rwaldron/tc39-notes/blob/master/es8/2017-03/mar-21.md#conclusionresolution-10) say "Stage will remain at stage 3", so that's where it will remain for the time being.

When there is a change, it will either fall back to stage 1, or be withdrawn, at which time I'll update the table.

Thanks for the contribution!

**Comment by leobalter** [comment]
@ljharb On my understanding, SIMD is not Stage 3 anymore. Pinging @littledan to confirm, if positive, we can fix this.

**Comment by ljharb** [comment]
@leobalter if you can link me in the notes to something later than what i linked to that says so, that'd be great, thanks

**Comment by littledan** [comment]
I think the best thing here would be to move SIMD.js to "Inactive proposals", with a link to the explainer, which documents the status. It seems like @ljharb is taking a pretty legalistic view of what happened--the proposal is effectively withdrawn, but could be proposed again by other people outside of the prior champion group want to bring this back up and convince those groups to change their mind. I posted the explanation as requested by the committee as a precondition for leaving Stage 3.

@ljharb Could you please cc me on PRs that relate to proposals I'm involved in, rather than just closing them? Thanks for noticing this, @leobalter .

**Comment by leobalter** [comment]
@ljharb, @littledan the communication has flaws and I'm sorry for this confusion.

The explanation is posted on the [repo's readme](https://github.com/tc39/ecmascript_simd) and, AFAICT, no one else volunteered to champion it so far.

This repo should reflects the withdrawn, and that's why I ping'ed @littledan.

Maybe it's reasonable if we bring this to the next meeting as an update that would help on clarification.

**Comment by littledan** [comment]
I don't know what there is to clarify. Is my written statement not clear?

**Comment by leobalter** [comment]
@littledan I believe the statement is clear, and I appreciate that. I believe the communication to the TC39 group can be improved through some sort of notification. Unfortunately, GitHub does not help telling users that Readme file is updated.

Maybe a quick update on the Reflector might help, maybe an update here - as suggested by this PR - is enough. IMHO, that's up to you, @littledan.

**Comment by ljharb** [comment]
Stage changes can only happen at meetings. As far as I'm aware, the last time it was discussed at a meeting, https://github.com/rwaldron/tc39-notes/blob/master/es8/2017-03/mar-21.md#conclusionresolution-10 was the resolution. Has the writeup been presented at a meeting, at which point it can become withdrawn or stage 1?

**Comment by littledan** [comment]
I sent the writeup to the Reflector to raise attention to it. At the last meeting, we didn't get through the whole agenda, and the next meeting may be a bit busy too. I also didn't make any new points in the writeup that weren't already explained in the last meeting. Let's move this to inactive, noting the people in that thread are caretaker replacement champions, and then new champions could go and bring things up when appropriate. It seems clear that this proposal is inactive, and that would accurately capture the resolution there. cc @wycats

**Comment by ljharb** [comment]
@littledan thanks, since that's not clear in the meeting notes, and only champions should ideally be making changes to their proposals in this repo, would you be able to make that PR?

**Comment by krainboltgreene** [comment]
Final question from me: For this situation should we completely remove from the table as suggested or instead use a "external link" icon like wikipedia that links to the removal notice?

**Comment by ljharb** [comment]
@krainboltgreene when the champion makes a PR for it, it would be completely removed from the active proposals table, and moved to the "inactive proposals" table.

**Comment by littledan** [comment]
@ljharb I've clarified things here; I don't see how it matters who makes the PR from now. If someone else makes a PR, I'll review it. If I have to make it, it'll probably wait a couple weeks.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This list contains only stage 1 proposals and higher that have not yet been with
 
 | 🚀 | Proposal                                                                                                    | Champion                           | Stage |
 |---|-------------------------------------------------------------------------------------------------------------|------------------------------------|-------|
-|   | [SIMD.JS - SIMD APIs](https://github.com/tc39/ecmascript_simd/)                                             | John McCutchan, Peter Jensen, Dan Gohman, Daniel Ehrenberg | 3 |
+|   | [SIMD.JS - SIMD APIs](https://github.com/tc39/ecmascript_simd/)                                             | John McCutchan, Peter Jensen, Dan Gohman, Daniel Ehrenberg | 0? |
```
Comment (chicoxyzzy) [inline README.md]: This proposal should be removed from this table completely. Also you should add it to https://github.com/tc39/proposals/blob/master/inactive-proposals.md

**Comment by ljharb** [comment]
I've made this change [here](35e0032d8b1cb5249a80629eabdd2c444acdea6a).

**Pull request closed without merging**